drivers/modem/modem_pin.c

```diff
@@ -22,7 +22,7 @@ int modem_pin_read(struct modem_context *ctx, u32_t pin)
 		return -ENODEV;
 	}
 
-	return gpio_pin_get_raw(ctx->pins[pin].gpio_port_dev,
```

Comment:
@pabigot I forget why we used the raw API here?

Reply:
Probably because the existing driver (sara-r4) has baked the logical state of the pin into the driver.  That is, asserting the reset pin in the sara-r4 driver sets the pin to physical state 0 instead of a logical state 1.
This issue arose when I was looking at using the sara-r4 driver for the Arduino MKR NB 1500.  On that board, there is an inversion of the signal between the mcu and the sara-r4 because of a level translator.  So, the sara-r4 driver will not work without a modification.  My goal is to make this a device tree setting for the board.  That is, you specify whether the pin is active high or active low in the device tree, not the driver.